pkg/cmd/scan.go

```diff
@@ -209,6 +209,7 @@ func scanRun(opts *pkg.ScanOptions) error {
 
 	err := remote.Activate(opts.To, opts.ProviderVersion, alerter, providerLibrary, remoteLibrary, scanProgress, resourceSchemaRepository, resFactory, opts.ConfigDir)
 	if err != nil {
+		WriteAlert(alerter)
```

Comment:
What is the usecase that require to write alerts here ?

I feel that we are using the actual alert component for a different usage and I'm not comfortable with this. This can lead to unexpected behaviors IMHO.